src/polish.c

```diff
@@ -296,8 +297,17 @@ OSQPInt polish(OSQPSolver* solver) {
   mred = form_Ared(work);
   if (mred < 0) {
     // Polishing failed
-    info->status_polish = -1;
-    return -1;
+    info->status_polish = OSQP_POLISH_FAILED;
+    return OSQP_POLISH_FAILED;
+  } else if (mred == 0) {
+    /* No active constraints, so skip polishing */
+    c_print("Polishing skipped - no active set detected at optimal point\n");
```

Comment:
"skipped" -> "not needed" is clearer I think.

Reply:
Change to "not needed" in most recent commit.